### PR TITLE
Add missing dconf OSPP Ansible remediations

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: linux_os/guide/system/software/gnome/gnome_login_screen/dconf_gnome_login_retries/ansible/shared.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+- name: "Enable the GNOME3 Login Smartcard Authentication"
```
Comment (jan-cerny) [inline linux_os/guide/system/software/gnome/gnome_login_screen/dconf_gnome_login_retries/ansible/shared.yml]: This configures number of login failures, not smartcards.

File: linux_os/guide/system/software/gnome/gnome_screen_locking/dconf_gnome_session_idle_user_locks/ansible/shared.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+
+- name: "Prevent user modification of GNOME Session idle-delayd"
```
Comment (jan-cerny) [inline linux_os/guide/system/software/gnome/gnome_screen_locking/dconf_gnome_session_idle_user_locks/ansible/shared.yml]: trailing d

File: linux_os/guide/system/software/gnome/gnome_login_screen/dconf_gnome_login_retries/rule.yml [hunk 3]
```diff
@@ -4,7 +4,20 @@ prodtype: rhel7,fedora
...
+description: |-
+    In the default graphical environment, the GNOME3 login
+    screen and be configured to restart the authentication process after
+    a configured number of attempts. This can be configured by setting
+    <tt>allowed-failures</tt> to <tt>3</tt> or less.
+    <br /><br />
+    To enable, add or edit <tt>allowed-failures</tt> to
+    <tt>/etc/dconf/db/gdm.d/00-security-settings</tt>. For example:
+    <pre>[org/gnome/login-screen]\nallowed-failures=3</pre>
```
Comment (jan-cerny) [inline linux_os/guide/system/software/gnome/gnome_login_screen/dconf_gnome_login_retries/rule.yml]: Could you remove also this `\n` ?

**Comment by redhatrises** [comment]
Thanks @jan-cerny. Fixed issues.

**Comment by jan-cerny** [comment]
Thank you.

**Pull request merged**